yaml_files/simphony_metadata.yml

```diff
@@ -444,38 +444,34 @@ CUDS_KEYS:
   ######################################################
   # Conditions on Boundaries
   ######################################################
-  DIRICHLET:
-    definition: Dirichlet boundary condition, specify the value the solutions takes on the boundary of the domain
+  VALUE_CONDITION:
+    definition: Boundary condition encountered in the solution of partial differential equations
     parent: CUBA.CONDITION
     models: [CUBA.CONTINUUM]
     CUBA.VARIABLE:
-      shape: (:)
+      shape: [1]
       default: []
-    CUBA.MATERIAL:
+    CUBA.VALUE:
       shape: (:)
+      default: 0.0
+    CUBA.MATERIAL:
+      shape: [1]
       default: []
 
+
+    
+  DIRICHLET:
+    definition: Dirichlet boundary condition to specify the value the solutions takes on the boundary of the domain.
+    parent: CUBA.VALUE_CONDITION
+
   NEUMANN:
```

Comment:
If we are not using Dirichlet/Neumann BC names, i think thaat equally also Neumann should be changes to for example GRADIENT_CONDITION